packages/server/src/api/controllers/user.js

```diff
@@ -63,7 +63,23 @@ exports.create = async function(ctx) {
   }
 }
 
-exports.update = async function() {}
+exports.update = async function(ctx) {
+  const db = new CouchDB(ctx.user.instanceId)
+  const user = ctx.request.body
+
+  const response = await db.put(user)
+  user._rev = response.rev
+
+  ctx.status = 200
+  ctx.body = {
```

Comment:
have a look at the `create` method, in this controller. 

`body` should just be the data returned, `message` should be `ctx.message =.... `

Also, we should not be returning the whole user object, as this will return the user's password hash... again.. see `create`

Reply:
Looking at this... should I just return the result from the put operation or do I make sure to return the user in the body?